test/atc_test.coffee

```diff
@@ -43,6 +43,21 @@ describe 'hubot-atc', ->
     room.robot.brain.data.locks = locks
 
   describe "applications", ->
+    describe "listing", ->
+      context "when there are none", ->
+        it 'responds with the proper message', ->
+          @room.user.say("duncan", "hubot application list").then =>
+            expect(lastMessage(@room)).to.match /No applications exist/
+
+      context "when there are already apps", ->
+        beforeEach ->
+          addApplication(@room, "foo")
+          addApplication(@room, "bar")
+
+        it 'responds with the proper message', ->
+          @room.user.say("duncan", "hubot application list").then =>
+            expect(lastMessage(@room)).to.match /applications available: bar, foo/
```

Comment:
NAB, but we can be more explicit here since [hubot-test-helper](https://github.com/mtsmfm/hubot-test-helper) gives us the ability to assert on the room's messages, for example:
```
expect(@room.messages).to.eql [
  ['alice', '@hubot hi']
  ['hubot', '@alice hi']
  ['bob',   '@hubot hi']
  ['hubot', '@bob hi']
]
```
would allow us to assert exactly what the room's state should be, rather than trying to regex a specific string

Reply:
Done, and done.